README.md

```diff
@@ -82,13 +82,13 @@ Good luck!
 
 Binaries for Linux compiled statically with the musl C library are available at this location:
 
-* [aarch64](http://repo.voidlinux.eu/static/xbps-static-latest.aarch64-musl.tar.xz)
-* [armv6hf](http://repo.voidlinux.eu/static/xbps-static-latest.armv6l-musl.tar.xz)
-* [i686](http://repo.voidlinux.eu/static/xbps-static-latest.i686-musl.tar.xz)
-* [x86\_64](http://repo.voidlinux.eu/static/xbps-static-latest.x86_64-musl.tar.xz)
-* [mips32](http://repo.voidlinux.eu/static/xbps-static-latest.mips-musl.tar.xz)
+* [aarch64](https://repo.voidlinux.eu/static/xbps-static-latest.aarch64-musl.tar.xz)
+* [armv6hf](https://repo.voidlinux.eu/static/xbps-static-latest.armv6l-musl.tar.xz)
+* [i686](https://repo.voidlinux.eu/static/xbps-static-latest.i686-musl.tar.xz)
+* [x86\_64](https://repo.voidlinux.eu/static/xbps-static-latest.x86_64-musl.tar.xz)
+* [mips32](https://repo.voidlinux.eu/static/xbps-static-latest.mips-musl.tar.xz)
 
-Check the *sha256* checksums with this file: http://repo.voidlinux.eu/static/sha256sums.txt
+Check the *sha256* checksums with this file: https://repo.voidlinux.eu/static/sha256sums.txt
```

Comment:
All of these change from `.eu` to `.org`

Reply:
You mean `https://alpha.de.repo.voidlinux.org/static/sha256sums.txt` (and similar?).